src/core/dataApiBuilder/dab.ts

```diff
@@ -101,7 +100,8 @@ async function getReleaseDataApiBuilderMetadata(): Promise<{ releaseMetadata: Da
   const responseMetadata = (await response.json()) as DataApiBuilderReleaseMetadata;
 
   if (Array.isArray(responseMetadata)) {
-    const releaseMetadata = responseMetadata.find((c) => c.releaseType === DATA_API_BUILDER_RELEASE_TAG);
+    // const releaseMetadata = responseMetadata.find((c) => c.version === DATA_API_BUILDER_LATEST_TAG); // If we want to proceed with downloading the latest tag
```

Comment:
should we also continue to check for releaseType as well, no?

Reply:
releaseType is released for all the released versions, so it can be added as an additional check but not useful imo.